### PR TITLE
EUI-4107-Document-type

### PR DESCRIPTION
**Pull request by ishOsw**
- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-4107

### Change description ###

Extend document types to pass into the MediaViewer

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/shared/services/document-management/document-management.service.ts [hunk 1]
```diff
@@ -50,14 +58,35 @@ export class DocumentManagementService {
       if (position === documentFieldValue.document_filename.length) {
         fileExtension = '';
       } else if (position >= 0) {
-        fileExtension = documentFieldValue.document_filename.slice(position + 1);
+        fileExtension = documentFieldValue.document_filename.slice(position + 1).toUpperCase();
```
Comment (adnanakgun) [inline src/shared/services/document-management/document-management.service.ts:61]: Hey @ishOsw,

The `.toUpperCase()` could be what is breaking this. In any case this is breaking the test. You need to come up with a solution that doesn't break what is already there.

Comment (adnanakgun) [reply src/shared/services/document-management/document-management.service.ts:61]: The test for `should return media viewer data for tif content types` in src/shared/services/document-management/document-management.service.spec.ts  specifically deals with the issue of case sensitivity.

**Review by Mo-Lala84 — DISMISSED** [review]
approved

File: src/shared/services/document-management/document-management.service.ts [hunk 2]
```diff
@@ -55,16 +63,37 @@ export class DocumentManagementService {
...
+      return DocumentManagementService.RTF;
     } else if (fileExtension.toLowerCase() === 'pdf') {
       return DocumentManagementService.PDF;
     } else {
-      console.warn(`Unknown content type with the file extension: ${fileExtension}`);
       return fileExtension;
     }
   }
 
   isImage(imageType: string) {
```
Comment (paul-graham) [inline src/shared/services/document-management/document-management.service.ts:83]: _Nitpick:_
Should have a return type of `boolean` and a `public`/`private` indicator. The same for all of the `isX` ones in here, actually...

**Review by paul-graham — COMMENTED** [review]
Separate discussion about removing the CSV type, which isn't supported by the API... but also this little nitpick if you have time for it. 🙂

File: src/shared/services/document-management/document-management.service.ts (hunk 2 above)
Comment (adnanakgun) [reply src/shared/services/document-management/document-management.service.ts:83]: You are right of course.

**Review by paul-graham — APPROVED** [review]
Excellent stuff. Thank you for making the requested changes, @adnanakgun, including my little nitpick!

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**